src/audio/google_rtc_audio_processing_mock.c

```diff
@@ -53,21 +100,46 @@ void GoogleRtcAudioProcessingFree(GoogleRtcAudioProcessingState *state)
 	}
 }
 
+int GoogleRtcAudioProcessingSetStreamFormats(GoogleRtcAudioProcessingState *const state,
+					     int capture_sample_rate_hz,
+					     int num_capture_input_channels,
+					     int num_capture_output_channels,
+					     int render_sample_rate_hz,
+					     int num_render_channels)
+{
+	SetFormats(state,
+		   capture_sample_rate_hz,
+		   num_capture_input_channels,
+		   num_capture_output_channels,
+		   render_sample_rate_hz,
+		   num_render_channels);
+	return 0;
+}
+
+int GoogleRtcAudioProcessingParameters(GoogleRtcAudioProcessingState *const state,
+				       float *capture_headroom_linear,
+				       float *echo_path_delay_ms)
+{
+	return 0;
+}
+
 int GoogleRtcAudioProcessingGetFramesizeInMs(GoogleRtcAudioProcessingState *state)
 {
-	return state->num_frames * 1000 / GOOGLE_RTC_AUDIO_PROCESSING_SAMPLE_RATE_HZ;
+	return state->num_frames *
+		GOOGLE_RTC_AUDIO_PROCESSING_MS_PER_SECOND /
+		GOOGLE_RTC_AUDIO_PROCESSING_SAMPLE_RATE_HZ;
 }
 
 int GoogleRtcAudioProcessingReconfigure(
-	GoogleRtcAudioProcessingState * const state, const uint8_t *const config,
-	int config_size)
+					GoogleRtcAudioProcessingState * const state, const uint8_t *const config,
+					int config_size)
```

Comment:
if you're aligning these parameters now, you could just put the first parameter one line above and move all other parameters onto the second line. Same below

Reply:
True. 
Done.